features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/BaseSwitchCompatMapperTest.kt

```diff
@@ -165,74 +211,22 @@ internal abstract class BaseSwitchCompatMapperTest : LegacyBaseWireframeMapperTe
         assertThat(resolvedWireframes).isEqualTo(fakeTextWireframes)
     }
 
-    @Test
+    @RepeatedTest(10)
```

Comment:
why are we repeating this test?

Reply:
because in the function forge generate random data, so I want to repeat the test so that  it can cover maximum the case